db/migrate/20230123163754_create_courses.rb

```diff
@@ -0,0 +1,13 @@
+class CreateCourses < ActiveRecord::Migration[7.0]
+  def change
+    create_table :courses, id: :uuid do |t|
+      t.string :title
```

Comment:
@jenlu33 do we need validation for any of these fields?

Reply:
@FerPerales I think the `title` should be unique with the `organization_id`, but I can add that in after creating an organization model.